src/main.rs

```diff
@@ -142,7 +142,8 @@ fn invite_from_ldap(
                         // println!("Invite response: {:?}", response);
                     }
                 } else {
-                    println!("Warning: Email field, {:?}, not found on user", mail_field);
+                    let user_uid = ldap_user.attrs.get("uid").and_then(|l| l.first()).unwrap();
```

Comment:
Rather than calling `unwrap()`, can we log print this line if we do have `uid` and then print the old message if there is no `uid`?